hummingbird/ml/operator_converters/sklearn/imputer.py

```diff
@@ -99,5 +101,6 @@ def convert_sklearn_missing_indicator(operator, device, extra_config):
     return MissingIndicator(operator.raw_operator, device)
 
 
+register_converter("SklearnImputer", convert_sklearn_simple_imputer)
```

Comment:
Is this only necessary for backward compatibility reasons?

Reply:
Yes. Because Imputer was deprecated in 0.22.